deployment/base/crdtmerge-deployment.yaml

```diff
@@ -0,0 +1,144 @@
+# https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
+apiVersion: v1
+kind: Service
+metadata:
+  name: crdtmerge
+  namespace: languagedepot
+  labels:
+    app: crdtmerge
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    app: crdtmerge
+  ports:
+  - name: http
+    protocol: TCP
+    port: 80
+
+---
+
+# https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: crdtmerge
+  namespace: languagedepot
+  labels:
+    app: crdtmerge
+spec:
+  selector:
+    matchLabels:
+      app: crdtmerge
+  strategy:
+    rollingUpdate:
+      maxSurge: 2
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    # https://kubernetes.io/docs/concepts/workloads/pods/#pod-templates
+    metadata:
+      labels:
+        app: crdtmerge
+    spec:
+      securityContext:
+        runAsGroup: 33
+        runAsUser: 33
+        runAsNonRoot: true
+      containers:
+      - name: crdtmerge
+        image: ghcr.io/sillsdev/lexbox-crdtmerge:develop
+        imagePullPolicy: IfNotPresent
+        # https://kubernetes.io/docs/concepts/configuration/manage-resources-containers
+        resources:
+          requests:
+            memory: 1500Mi
+          limits:
+            memory: 2400Mi
+        startupProbe:
+          httpGet:
+            port: 80
+            path: /api/healthz
+          failureThreshold: 30
+          periodSeconds: 10
+        ports:
+          - containerPort: 80
+
+        volumeMounts:
+        - name: crdtmerge
+          mountPath: /var/lib/crdtmerge
+
+        env:
+          - name: DOTNET_URLS
+            value: http://0.0.0.0:80
+          - name: ASPNETCORE_ENVIRONMENT
+            valueFrom:
+              configMapKeyRef:
+                name: app-config
+                key: environment-name
+          - name: K8S_NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: K8S_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: POSTGRES_DB
+            valueFrom:
+              secretKeyRef:
+                  key: POSTGRES_DB
+                  name: db
+          - name: POSTGRES_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                  key: POSTGRES_PASSWORD
+                  name: db
+          - name: DbConfig__LexBoxConnectionString
+            value: Host=db;Port=5432;Username=postgres;Password=$(POSTGRES_PASSWORD);Database=$(POSTGRES_DB)
+          - name: SendReceiveConfig__ProjectStorageRoot
+            value: /var/lib/crdtmerge/projects
+          - name: HgConfig__SendReceiveDomain
+            valueFrom:
+              configMapKeyRef:
+                name: app-config
+                key: hg-domain
```

Comment:
this app-config.hg-domain is actually going to be the public domain used for S&R, we actually just want to use lexbox:5158 to access lexbox directly inside the cluster

Reply:
That's a copy-and-paste-and-edit leftover; I removed it in commit 62e809c1. The `SendReceiveConfig__LexboxUrl` value just below is the one we're actually using (and it is indeed set to `http://lexbox:5158/` already).

Reply:
Just rebased on top of `develop` to solve merge conflicts; that's commit efd2665b now.